src/hackathons-collector/hackathons-collector.js

```diff
@@ -1,81 +1,78 @@
+import logo from './assets/logo.svg';
+import akai from './assets/akai-logo.svg';
 
-export default class HackathonsCollector extends Component {
-  constructor() {
-    super();
-    this.state = {
-      filterWeekends: false,
-      allEvents: events
-    };
-  }
-  render() {
-    const previousEvents = this.state.allEvents
-      .filter(event => event.date <= new Date())
-      .filter(event => {
-        return !this.state.filterWeekends || event.date.getDay() > 5;
-      })
-      .sort((a, b) => a.date - b.date)
-      .reverse()
-      .splice(0, 3);
-    const events = this.state.allEvents
-      .filter(event => event.date > new Date())
-      .filter(event => {
-        return !this.state.filterWeekends || event.date.getDay() > 5;
-      })
-      .sort((a, b) => a.date - b.date);
-    return (
-      <div>
-        <Header>
-          <Logo src={logo} />
-          <H1>Hackathons Collector</H1>
-        </Header>
-        <Filter
-          value={this.state.filterWeekends}
-          setValue={newValue => {
-            this.setState({ filterWeekends: newValue });
-          }}
-        >
-          Only weekend events
-        </Filter>
-        <Main>
-          <Section>
-            <H2>Incoming hackathons:</H2>
-            <Listing list={events} />
-          </Section>
-          <Section>
-            <H2>Previous hackathons:</H2>
-            <Listing list={previousEvents} disabled={true} />
-          </Section>
-        </Main>
-        <Footer>
-          Created by{" "}
-          <a href="https://akai.org.pl/" title="AKAI">
-            <img src={akai} alt="AKAI" height="12" />
-          </a>
-        </Footer>
-      </div>
-    );
-  }
-}
+const HackathonsCollector = ({ events }) => {
+  const [filterWeekends, changeFilterWeekends] = useState(false);
+  const [allEvents] = useState(events);
+
+  const previousEvents = allEvents
+    .filter(({ date }) => date <= new Date())
+    .filter(({ date }) => {
+      return !filterWeekends || date.getDay() > 5;
+    })
+    .sort((a, b) => a.date - b.date)
+    .reverse()
+    .splice(0, 3);
+
+  const updatedEvents = allEvents
+    .filter(({ date }) => date > new Date())
+    .filter(({ date }) => {
+      return !filterWeekends || date.getDay() > 5;
+    })
+    .sort((a, b) => a.date - b.date);
+
+  return (
+    <div>
+      <Header>
+        <Logo src={logo} />
+        <H1>Hackathons Collector</H1>
+      </Header>
+      <Filter
+        value={filterWeekends}
+        setValue={(newValue) => {
+          changeFilterWeekends(newValue);
+        }}
+      >
+        Only weekend events
+      </Filter>
+      <Main>
+        <Section>
+          <H2>Incoming hackathons:</H2>
+          <Listing list={updatedEvents} />
+        </Section>
+        <Section>
+          <H2>Previous hackathons:</H2>
+          <Listing list={previousEvents} disabled={true} />
+        </Section>
+      </Main>
+      <Footer>
+        Created by{' '}
```

Comment:
wdyt about
```suggestion
        <span>Created by</span>
```
?

Reply:
Done!

Reply:
<img width="130" alt="Screenshot 2020-09-26 at 20 02 48" src="https://user-images.githubusercontent.com/20808724/94347314-4a036280-0033-11eb-8ce9-620fe597f22b.png">
welp, without space it does not look that great! I'd recommend to add some margin to the logo in footer, for example
```javascript
const Footer = styled("footer")`
  font-size: small;
  max-width: 600px;
  margin: 20px auto;
  text-align: center;

  a {
    margin-left: 4px;
  }
`;
```

Reply:
Done!